google/cloud/google_cloud_cpp_common.cmake

```diff
@@ -214,6 +214,42 @@ install(
     DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/google_cloud_cpp_common"
     COMPONENT google_cloud_cpp_development)
 
+# Create a header-only library for the mocks. We use a CMake `INTERFACE` library
+# for these, a regular library would not work on macOS (where the library needs
+# at least one .o file). Unfortunately INTERFACE libraries are a bit weird in
```

Comment:
I think this comment is now out of date?

Reply:
At the moment, it is a header only library.

Reply:
yes, but you are not using the absolute paths anymore?

Reply:
ah, thanks. I did not read the comment closely.